test/e2e/scripts/prow_run_smoke_test.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
+
+# This script automates the complete deployment and validation of the MaaS platform on OpenShift
+
+set -euo pipefail
+
+# Configuration
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROJECT_ROOT="$(cd "$SCRIPT_DIR/../../.." && pwd)"
```

Comment:
It might not survive refactoring. To future-proof we can use some other way of finding the root folder.

Maybe something like this?  https://github.com/opendatahub-io/kserve/blob/e585d3953efb855fb60737e0ec52f5a129f55ce1/test/scripts/openshift-ci/common.sh#L17-L37